core-schema/src/main/resources/sql-schema/V1__Create_initial_schema.sql

```diff
@@ -47,10 +47,20 @@ CREATE TABLE prefs_flags (
   FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE
 );
 
-CREATE TABLE prefs_blockedusers (
+CREATE TABLE friends_blocked_users (
   user_id INT NOT NULL,
   blocked_user_id INT NOT NULL,
   FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
   FOREIGN KEY (blocked_user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
-  INDEX (user_id)
+  INDEX (user_id),
+  UNIQUE (user_id,blocked_user_id)
+);
+
+CREATE TABLE friends_friended_users (
+  user_id INT NOT NULL,
+  friend_user_id INT NOT NULL,
+  FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
+  FOREIGN KEY (friend_user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
+  INDEX (user_id),
```

Comment:
Same here. You can remove this index.

It's my bad for showing you the wrong example the first time I made the schema. I had just learned indexes then, good thing I know how they work now.

Reply:
Got it: You learned indexes fast!